api/lib/domain/models/UserDetailsForAdmin.js

```diff
@@ -43,6 +45,14 @@ class UserDetailsForAdmin {
     this.hasBeenAnonymised = hasBeenAnonymised;
     this.hasBeenAnonymisedBy = hasBeenAnonymisedBy;
     this.updatedAt = updatedAt;
+    this.anonymisedByFirstName = anonymisedByFirstName;
+    this.anonymisedByLastName = anonymisedByLastName;
+  }
+
+  get anonymisedByFullName() {
+    return this.anonymisedByFirstName && this.anonymisedByLastName
+      ? `${this.anonymisedByFirstName} ${this.anonymisedByLastName}`
+      : null;
   }
```

Comment:
Si le nom et prénom sont obligatoires alors ne prends pas en compte ma suggestion.

Dans le cas contraire, si l'utilisateur n'a que le nom ou le prénom la chaine de caractères pourrait être `Alain ` ou ` Verse`. Pour éviter cette espace, on pourrait envisager ceci 

```suggestion
  get anonymisedByFullName() {
    const fullName = [];
    if (this.anonymisedByFirstName) fullName.push(this.anonymisedByFirstName);
    if (this.anonymisedByLastName) fullName.push(this.anonymisedByLastName);
    return fullName.join(' ');
  }
```

Reply:
Contre proposition plus simple :

```suggestion
  get anonymisedByFullName() {
    return this.anonymisedByFirstName && this.anonymisedByLastName
      ? `${this.anonymisedByFirstName} ${this.anonymisedByLastName}`.trim()
      : null;
  }
```

Reply:
Il faut demander si c'est obligatoire à Emmy mais dans les faits, tous les agents Pix ont un nom et prénom.
( voir sur metabase `SELECT "users"."firstName", "users"."lastName" FROM "users" INNER JOIN "pix-admin-roles"  ON "users"."id" = "pix-admin-roles"."userId" ;`)

Reply:
Effectivement, d'ailleurs :

![Capture d’écran 2023-01-05 à 15 20 50](https://user-images.githubusercontent.com/5627688/210801749-310e8c1a-3f55-4032-bef4-7682988fd290.png)